README.md

```diff
@@ -174,59 +131,29 @@ metal_vrf = [
   },
 ]
 metrovlan_ids = [
-  1006,
-  1007,
+  1008,
+  1009,
 ]
 server_name = [
   "mymetal-node-1",
   "mymetal-node-2",
 ]
-ssh_private_key = "/Users/johndoe/myterraform/myvrf/Cloudinit/ssh-key-wieyh"
-virtual_connection_primary = [
-  {
-    "connection_id" = "06726413-c565-4173-82be-9a9562b9a69b"
-    "customer_ip" = "169.254.100.2"
-    "description" = "Primary Virtual Circuit"
-    "id" = "acb4f4cf-9c96-424c-bb6f-51bda36b17a5"
-    "md5" = ""
-    "metal_ip" = "169.254.100.1"
-    "name" = "virtual_connection_pri"
-    "nni_vlan" = 999
-    "nni_vnid" = 999
-    "peer_asn" = 100
-    "port_id" = "2a027f9f-dae6-454b-a783-2a25c678f506"
-    "project_id" = "81666c08-3823-4180-832f-1ce1f13e1662"
-    "speed" = "0"
-    "status" = "active"
-    "subnet" = "169.254.100.0/30"
-    "tags" = tolist(null) /* of string */
-    "vlan_id" = tostring(null)
-    "vnid" = 0
-    "vrf_id" = "78034f87-5733-494f-b288-46c8a18da3f4"
-  },
-]
-virtual_connection_secondary = [
-  {
-    "connection_id" = "06726413-c565-4173-82be-9a9562b9a69b"
-    "customer_ip" = "169.254.100.10"
-    "description" = "Secondary Virtual Circuit"
-    "id" = "16db378e-923d-4c52-a8de-27b507e32dcc"
-    "md5" = ""
-    "metal_ip" = "169.254.100.9"
-    "name" = "virtual_connection_sec"
-    "nni_vlan" = 999
-    "nni_vnid" = 999
-    "peer_asn" = 100
-    "port_id" = "963471e8-c815-4055-93b6-74092227d65c"
-    "project_id" = "81666c08-3823-4180-832f-1ce1f13e1662"
-    "speed" = "0"
-    "status" = "active"
-    "subnet" = "169.254.100.8/30"
-    "tags" = tolist(null) /* of string */
-    "vlan_id" = tostring(null)
-    "vnid" = 0
-    "vrf_id" = "78034f87-5733-494f-b288-46c8a18da3f4"
-  },
-]
+ssh_private_key = "/Users/larryma/myterraform/myvrf/0801_2022/terraform-equinix-metal-vrf/ssh-key-711c9"
```

Comment:
Did you want to preserve the path name that's in the repo today?

Reply:
Good point and thanks! changed it a moment ago